### ISSUE TITLE
Would it be possible to evaluate scene graph from visual genome dataset

### ISSUE THREAD

**Issue by xiaofeiwu** [peteanderson80/SPICE#7]
Hi Peter,

Thanks for the great work. I wonder would it be possible to evaluate scene graph from visual genome dataset. Is there a demo?

**Comment by peteanderson80** [comment]
Yes it would be possible. There is no demo because I haven't ever tried it. I'd be really interested to see how it works. You would need to modify the code somewhat. Basically, instead of loading the reference scene graphs by [parsing the reference captions](https://github.com/peteanderson80/SPICE/blob/3e5de7a33c0a512427a7c0cf3bea2262d83fd18b/src/main/java/edu/anu/spice/SpiceScorer.java#L109), you would need to implement another method to load reference scene graphs from the VG annotations.
